src/components/ProfilingToolMenu/index.native.tsx

```diff
@@ -0,0 +1,179 @@
+        if (shouldProfiling) {
+            startProfiling();
+        } else {
+            stop();
+        }
+        toggleProfileTool();
+        return () => {
+            stop();
+        };
+    }, [isProfilingInProgress, stop]);
+
+    const getAppInfo = useCallback(
+        () =>
+            JSON.stringify({
+                appVersion: pkg.version,
+                environment: CONFIG.ENVIRONMENT,
+                platform: getPlatform(),
+                totalMemory: formatBytes(totalMemory, 2),
+                usedMemory: formatBytes(usedMemory, 2),
+            }),
+        [totalMemory, usedMemory],
+    );
+
+    useEffect(() => {
+        if (!pathIOS) {
+            return;
+        }
+
+        // eslint-disable-next-line @lwc/lwc/no-async-await
+        const rename = async () => {
+            const newFileName = `Profile_trace_for_${pkg.version}.cpuprofile`;
+            const newFilePath = `${RNFS.DocumentDirectoryPath}/${newFileName}`;
+
+            try {
+                const fileExists = await RNFS.exists(newFilePath);
+                if (fileExists) {
+                    await RNFS.unlink(newFilePath);
+                    Log.hmmm('[ProfilingToolMenu] existing file deleted successfully');
+                }
+            } catch (error) {
+                const typedError = error as Error;
+                Log.hmmm('[ProfilingToolMenu] error checking/deleting existing file: ', typedError.message);
+            }
+
+            // Copy the file to a new location with the desired filename
+            await RNFS.copyFile(pathIOS, newFilePath)
+                .then(() => {
+                    Log.hmmm('[ProfilingToolMenu] file copied successfully');
+                })
+                .catch((error) => {
+                    Log.hmmm('[ProfilingToolMenu] error copying file: ', error);
+                });
+
+            setSharePath(newFilePath);
+        };
+
+        rename();
+    }, [pathIOS]);
+
+    const onDownloadProfiling = useCallback(() => {
+        // eslint-disable-next-line @lwc/lwc/no-async-await
+        const shareFiles = async () => {
+            try {
+                // Define new filename and path for the app info file
+                const infoFileName = `App_Info_${pkg.version}.json`;
+                const infoFilePath = `${RNFS.DocumentDirectoryPath}/${infoFileName}`;
+                const actualInfoFile = `file://${infoFilePath}`;
+
+                await RNFS.writeFile(infoFilePath, getAppInfo(), 'utf8');
+
+                const shareOptions = {
+                    urls: [`file://${sharePath}`, actualInfoFile],
+                };
+
+                await Share.open(shareOptions);
+            } catch (error) {
+                console.error('Error renaming and sharing file:', error);
+            }
+        };
+        shareFiles();
+    }, [getAppInfo, sharePath]);
+
+    return (
+        <>
+            <Text
+                style={[styles.textLabelSupporting, styles.mb4]}
```

Comment:
Not a blocker but bottom section title is so closer to top section

<img width="325" alt="Screenshot 2024-02-22 at 12 56 35 AM" src="https://github.com/Expensify/App/assets/108292595/0fb26ac5-7030-4bca-bef0-2e5dc5adf20e">